generators/base-application/support/prepare-field.js

```diff
@@ -254,6 +254,8 @@ function _derivedProperties(field) {
     fieldValidationUnique: validationRules.includes(UNIQUE),
     fieldValidationMinBytes: validationRules.includes(MINBYTES),
     fieldValidationMaxBytes: validationRules.includes(MAXBYTES),
+    fieldDefaultValueDefined: field.defaultValue !== undefined,
+    fieldHasAnyDefaultValue: field.defaultValue !== undefined || field.defaultValueComputed !== undefined,
```

Comment:
`fieldHasAnyDefaultValue` is only used in liquibase generator, not used in templates (the only usage I think it can be removed).
it can be extracted to a method in support/internal to don't require a recalculation in case of defaultValue change.

Reply:
Same thing here. We have it in three places, so I wanted to avoid duplicating the conditions in various places...

Reply:
OK, I think, I see what you want. Moving this and the other into the liquibase generator as it is only used in there...

Reply:
I think, it is alright now